modules/test/playwright/tests/layout-set-prototype-web/main/siteTemplatesWithWebContent.spec.ts

```diff
@@ -437,3 +439,109 @@ testWithPrivatePages(
 		);
 	}
 );
+
+test(
+	'Page links on sites which were created in site templates should redirect correctly to other pages.',
+	{tag: ['@LPD-46415']},
+	async ({
+		apiHelpers,
+		applicationsMenuPage,
+		page,
+		pageEditorPage,
+		pagesAdminPage,
+		productMenuPage,
+		sitesPage,
+		uiElementsPage,
+		webContentDisplayPage,
+	}) => {
+		const siteTemplateName: string = 'Template-' + getRandomString();
+
+		const layoutSetPrototype = await createSiteTemplate({
+			apiHelpers,
+			page,
+			productMenuPage,
+			templateName: siteTemplateName,
+		});
+
+		apiHelpers.data.push({
+			id: layoutSetPrototype.layoutSetPrototypeId,
+			type: 'layoutSetPrototype',
+		});
+
+		await productMenuPage.goToPages();
+
+		const page1Name = getRandomString();
+		await pagesAdminPage.createNewPage({
+			name: page1Name,
+		});
+		const page2Name = getRandomString();
+		await pagesAdminPage.createNewPage({
+			name: page2Name,
+		});
+
+		const basicWebContentStructureId =
+			await getBasicWebContentStructureId(apiHelpers);
+
+		const webContentName = getRandomString();
+
+		const site =
+			await apiHelpers.headlessAdminUser.getSiteByFriendlyUrlPath(
+				`Template-${layoutSetPrototype.layoutSetPrototypeId}`
+			);
+
+		const baseURL = liferayConfig.environment.baseUrl;
+		const cleanBaseURL = baseURL.replace('http://', '');
```

Comment:
maybe we can change from `cleanBaseURL` to something like `baseURLNoProtocol`? 🤔